scripts/test-check-packages.sh

```diff
@@ -68,13 +71,15 @@ if [ "${PACKAGE_TEST_TYPE:-other}" == "with-logstash" ]; then
   echo "stack.logstash_enabled: true" >> ~/.elastic-package/profiles/logstash/config.yml
 fi
 
-# Update the stack
-elastic-package stack update -v
+if [[ "${SERVERLESS}" != "true" ]]; then
```

Comment:
When testing with Serverless, Elastic stack is just started once at the beginning of the pipeline step. That's why here it is skipped.

Reply:
Could we also add a comment here mentioning this?

Reply:
Or maybe we can extract the common logic to check packages, and follow the same approach as in serverless in all targets? For a future refactor in any case.

Reply:
Added a comment for now in this script, here and also in the `cleanup` function.